recipes/default.rb

```diff
@@ -35,11 +18,20 @@
 end
 
 # install mimic
-python_pip path do
-  action :install
-  virtualenv venv
-  if is_dev
-    options "-e"
+if node['mimic']['dev']
+  bash 'install provided mimic' do
+    code <<-CODE
+      source #{venv}/bin/activate
+      cd #{path}
```

Comment:
Only using this variable once, I'm not sure it needs to be set in a local variable but whatever.